test/lib/docker.test.ts

```diff
@@ -533,11 +533,25 @@ test("pullCli", async (t) => {
 
   t.test("with args", async (t) => {
     await docker.pullCli(targetImage, { platform: "linux/arm64/v8" });
-    const subProcessArgs = stub.getCall(0).args;
+    const subProcessArgs = stub.getCall(1).args;
     t.same(
       subProcessArgs,
       ["docker", ["pull", "--platform=linux/arm64/v8", targetImage]],
       "args passed to subProcess.execute as expected",
     );
   });
+
+  t.test(
+    "with platform arg, when no experimental features enabled",
+    async (t) => {
+      stub.resolves({ stdout: "Experimental: false" });
+
+      t.rejects(
```

Comment:
I think we should `await t.rejects(` since it returns a Promise

Reply:
100%, thanks!